sections/errorhandling/documentingusingswagger.md

```diff
@@ -1,8 +1,8 @@
-# Document API errors using Swagger or GraphQL
+# Document API errors using OpenAPI Specification (eka Swagger) or GraphQL
 
 ### One Paragraph Explainer
 
-REST APIs return results using HTTP status codes, it’s absolutely required for the API user to be aware not only about the API schema but also about potential errors – the caller may then catch an error and tactfully handle it. For example, your API documentation might state in advance that HTTP status 409 is returned when the customer name already exists (assuming the API register new users) so the caller can correspondingly render the best UX for the given situation. Swagger is a standard that defines the schema of API documentation offering an eco-system of tools that allow creating documentation easily online, see print screens below
+REST APIs return results using HTTP status codes, it’s absolutely required for the API user to be aware not only about the API schema but also about potential errors – the caller may then catch an error and tactfully handle it. For example, your API documentation might state in advance that HTTP status 409 is returned when the customer name already exists (assuming the API register new users) so the caller can correspondingly render the best UX for the given situation. OpenAPI (eka Swagger) is a standard that defines the schema of API documentation offering an eco-system of tools that allow creating documentation easily online, see print screens below

```

Comment:
Shouldn't it be (aka Swagger)?

"aka" stands for "also known as", what "eka" means?

Reply:
Earlier known as

Reply:
To me it's known as `earliest know appearance` and I've not seen it used often.
I think we should go with something longer like "formerly Swagger" so that more people can understand 🙂

Reply:
I just thought that it's a TYPO, now that I see that it's a valid phrase - I'm good with this or whatever you land on